### ISSUE TITLE
pylint: Instance of 'tuple' has no 'extend' member (no-member)

### ISSUE THREAD

**Issue by aogier** [saltstack/salt#43086]
when developing on another branch I saw some errors that pylint indeed partly confirmed:

```shell
$ pylint --rcfile=.pylintrc salt/modules/groupadd.py
...
E: 55, 8: Instance of 'str' has no 'extend' member (no-member)
E: 75, 8: Instance of 'tuple' has no 'extend' member (no-member)
E:146, 8: Instance of 'tuple' has no 'extend' member (no-member)
E:179,12: Instance of 'tuple' has no 'extend' member (no-member)
E:183,12: Instance of 'tuple' has no 'extend' member (no-member)
...
```
This probably could get a better coverage in tests, I'll try to track it down and then fix it !

**Comment by garethgreenaway** [comment]
@aogier Thanks for the report!

**Comment by aogier** [comment]
closed in fc587f784a15b0c3f3dcf24767b776c9f7f1c3a8
